src/ast/ast_build_op_contexts.c

```diff
@@ -317,13 +313,13 @@ AST_CreateContext AST_PrepareCreateOp(GraphContext *gc, RecordMap *record_map, A
 			QueryGraph_AddPath(gc, ast, qg, path);
 
 			uint path_elem_count = cypher_ast_pattern_path_nelements(path);
-			for(uint j = 0; j < path_elem_count; j ++) {
+			for(uint k = 0; k < path_elem_count; k ++) {
```

Comment:
How have we missed this?!?

Reply:
Ugh, I know. I can't believe that this type of bug doesn't trigger a compiler warning?